### PR TITLE
Issue 008

### PR DESCRIPTION
**Pull request by Lkubok**
Merged with issue-007. Sorry for that, but it was easier to me to check what is going on...

So, there is a problem QuoteForm.jsx . in handleSubmit in the part that should update the quote something is going wrong. 

when I try tu update with PostMan with body like : 

````
{
	"id": "5d3d8a409592c64296348eff",
	"quote":{
	"quoteAuthor": "Margaret Sangsterrfdsfdsfrr",
    "lang": "eng",
    "insertAuthor": "admfdsfsdfin",
    "quote": "self-complacency is fatal to progress."
	}

}
````
it works. But with data from this handleSubmit it doesn't . Any simple solution ?

### CODE REVIEW

**Comment by tobob** [comment]
@Lkubok you did use 'post' instead of 'put' making request

**Comment by tobob** [comment]
problem is fixed, however, when I update a quote form is repopulated with an old version of quote

**Comment by Lkubok** [comment]
Problem with repopulation of Form solved. After axios is done with putting the data to server, a push is made on browserHistory.

File: src/components/QuoteForm/QuoteForm.jsx [hunk 1]
```diff
@@ -91,40 +98,47 @@ const myFormik = withFormik({
...
         });
     }
   }
 });
 
-export default compose(
-  connect(mapStateToProps),
-  myFormik
-)(QuoteForm);
+export default withRouter(
```
Comment (tobob) [inline src/components/QuoteForm/QuoteForm.jsx]: withRouter goes under compose

Comment (tobob) [reply src/components/QuoteForm/QuoteForm.jsx]:
```
  compose(
    withRouter,
    connect(
      mapStateToProps,
      mapDispatchToProps
    ),
    myFormik
  )(QuoteForm)
```

File: src/components/QuoteForm/QuoteForm.scss [hunk 2]
```diff
@@ -32,7 +32,7 @@
     resize: none;
     height: 100px;
   }
-  button {
```
Comment (tobob) [inline src/components/QuoteForm/QuoteForm.scss]: nice!

**Comment by tobob** [comment]
@Lkubok one small change, please fix it and merge it,
maybe you can add some small tests?

**Pull request merged**